### PR TITLE
Add rtrott to Moderation and report aliases

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

File: iojs.org/aliases.json [hunk 1]
```diff
@@ -158,7 +159,8 @@
       "ljharb@gmail.com",
       "othiym23@gmail.com",
       "ryanharrisonlewis@gmail.com",
-      "shelley.vohr@gmail.com"
+      "shelley.vohr@gmail.com",
+      "rtrott@gmail.com"
```
Comment (sam-github) [inline iojs.org/aliases.json]: you could alphabetize, looks like the others are

Comment (Trott) [reply iojs.org/aliases.json]: Ha! I thought I did. Whoops. Will fix.

File: iojs.org/aliases.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
       "ljharb@gmail.com",
       "othiym23@gmail.com",
       "ryanharrisonlewis@gmail.com",
+      "rtrott@gmail.com",
       "shelley.vohr@gmail.com"
```
Comment (Trott) [inline iojs.org/aliases.json]:
```suggestion
      "rtrott@gmail.com",
      "ryanharrisonlewis@gmail.com",
      "shelley.vohr@gmail.com"
```

File: iojs.org/aliases.json (hunk 1 above)
Comment (Trott) [inline iojs.org/aliases.json]:
```suggestion
      "rtrott@gmail.com",
      "ryanharrisonlewis@gmail.com",
      "shelley.vohr@gmail.com"
```

Comment (Trott) [inline iojs.org/aliases.json]:
```suggestion
      "rtrott@gmail.com",
      "ryanharrisonlewis@gmail.com",
      "shelley.vohr@gmail.com"
```

**Comment by rvagg** [comment]
@mhdawson you available to handle this? I don't normally like to jump on things like this where I have zero context but you probably do.

**Comment by mhdawson** [comment]
Deployed

**Pull request merged**